testnet.template

```diff
@@ -71,7 +71,9 @@ wcmd () {
 
 cacmd () {
     programs/cleos/cleos  --wallet-url $wdurl --url http://$bioshost:$biosport system newaccount --transfer --stake-net "10000000.0000 "$bioscurrencysymbol --stake-cpu "10000000.0000 "$bioscurrencysymbol  --buy-ram "10000000.0000 "$bioscurrencysymbol eosio $* >> $logfile 2>&1
+    sleep 1 # tmp fix until this is part of python regression test framework
```

Comment:
Maybe call out that it will be waiting for a transaction in block, when it is made available.